src/modules/temporary-file-check/file.service.ts

```diff
@@ -0,0 +1,25 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { MAX_FILE_SIZE_BYTES } from '@ukef/constants';
+import { DtfsStorageFileService } from '@ukef/modules/dtfs-storage/dtfs-storage-file.service';
+
+import { GetFileSizeRequestItem } from './dto/get-file-size-request.dto';
+import { GetFileSizeResponse } from './dto/get-file-size-response.dto';
+
+@Injectable()
+export class FileService {
```

Comment:
Out of interest is this temporary ie going to be removed later, or temporary ie checking for temporary files? Assuming the former with the naming of this service compared to the folder

Reply:
The former, as in going to be removed later